github/events/discussions.py

```diff
@@ -0,0 +1,102 @@
+import html
+
+from data_types.discussion import Discussion
+from data_types.repository import Repository
+from data_types.user import User
+from .base import EventBase
+
+
+class EventDiscussions(EventBase):
+
+    def __init__(self, sdk):
+        super(EventDiscussions, self).__init__(sdk)
+        self.discussion = None
+        self.repository = None
+        self.sender = None
+        self.sdk = sdk
+
+    """
+    DiscussionsEvent
+
+    Triggered when an discussion is created.
+
+    https://developer.github.com/v3/activity/events/types/#discussionevent
+    """
+
+    async def process(self, payload, chat):
+        """
+        Processes Discussions event
+        :param payload: JSON object with payload
+        :param chat: current chat object
+        :return:
+        """
+
+        self.sdk.log("Discussions event payload taken")
+
+        try:
+            self.discussion = Discussion(payload['discussion'])
+            self.repository = Repository(payload['repository'])
+            self.sender = User(payload['sender'])
+
+        except Exception as e:
+            self.sdk.log('Cannot process DiscussionsEvent payload because of {}'.format(e))
+
+        action = payload['action']
+
+        available_actions = {
+            'created': self.created,
+            'deleted': self.deleted,
+        }
+
+        if action not in available_actions:
+            self.sdk.log('Unsupported Discussions action: {}'.format(action))
+            return
+
+        # call action handler
+        await available_actions[action](chat['chat'], payload)
```

Comment:
won't it be an exception if there is no defined `available_actions`?
Maybe to use a check like `if action in available_actions:`?